gubernator/view_base.py

```diff
@@ -57,14 +59,33 @@
 jinja_filters.register(JINJA_ENVIRONMENT.filters)
 
 
-class RenderingHandler(webapp2.RequestHandler):
+class BaseHandler(webapp2.RequestHandler):
     """Base class for Handlers that render Jinja templates."""
     def __init__(self, *args, **kwargs):
-        super(RenderingHandler, self).__init__(*args, **kwargs)
+        super(BaseHandler, self).__init__(*args, **kwargs)
         # The default deadline of 5 seconds is too aggressive of a target for GCS
         # directory listing operations.
         urlfetch.set_default_fetch_deadline(60)
 
+    # This example code is from:
+    # http://webapp2.readthedocs.io/en/latest/api/webapp2_extras/sessions.html
+    def dispatch(self):
+        # pylint: disable=attribute-defined-outside-init
```

Comment:
The link to the webapp2 documentation is supposed to provide the explanation. Explaining the dispatch method and webapp2 middleware seems out scope.